histomicstk/features/ComputeFSDs.py

```diff
@@ -0,0 +1,179 @@
+import collections
+import numpy as np
+from skimage.segmentation import find_boundaries
+
+
+def ComputeFSDs(Label, K=128, Fs=6):
+    """
+    Calculates `Fourier shape descriptors` from a label mask.
+
+    Parameters
+    ----------
+    Label : array_like
+        A M x N label mask.
```

Comment:
We should be cautious about repeating calculations. Finding contours from a mask can be expensive and should be done once. Perhaps we can generate contours prior to feature extraction and pass them as a parameter?